content/faq/_index.md

```diff
@@ -149,3 +149,53 @@ you need and us making them available for you.
 
 In the mean time we are solving these requests one by one, so [please reach out
 to us](#how-can-i-contact-you).
+
+### A command failed in packit-service: how do I reproduce it locally?
+
+We don't have an end-to-end solution to this, yet.
+
+In the meantime, you can pull our production sandbox image
+(`docker.io/usercont/sandcastle:prod`) and run the command inside. As an
+example, this is how we were debugging build problems with anaconda:
+
+1. Clone the upstream git repo.
+
+2. Pull the sandbox image:
+
+```
+$ docker pull docker.io/usercont/sandcastle:prod
+```
+
+3. Launch the container and bind-mount the upstream project inside:
+
+```
+$ docker run -ti --rm -v $PWD:/anaconda docker.io/usercont/sandcastle:prod bash
+```
+
+4. Run commands of your choice:
```

Comment:
It would be nice to have an image with both sandcastle dependencies and packit installed to be able to run `packit srpm` there as well.

Reply:
Seems that packit is already installed there.

Reply:
Ah. Good to know..;)